package.json

```diff
@@ -243,5 +242,10 @@
     "**/@uirouter/angularjs": "1.0.11",
     "**/css-loader": "5",
     "**/moment": "^2.21.0"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
   }
```

Comment:
remove these lines

Reply:
this is currently on develop, I don't think my problem with husky is because of this

Reply:
right, sorry